packages/fxa-settings/src/pages/Index/container.tsx

```diff
@@ -158,36 +172,83 @@ export const IndexContainer = ({
         return getHandledError(error);
       }
     },
-    [authClient, handleSuccess, isWebChannelIntegration]
+    [authClient, handleSuccess, integration]
   );
 
+  // If there is a suggested email, try to direct to either signin or signup
+  // If no account exists for the suggested email, the same email validation steps
+  // will be run as if the email was submitted manually.
+  // RP-provided 'email' query param should take precedence, followed by 'login_hint'
+  // then previous accounts.
+  const suggestedEmail =
```

Comment:
Not a big deal, but curious if there is a reason these lines moved?

Reply:
Not an essential change, but moved to group all the code related to suggested email